src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -468,6 +468,12 @@ describe('DetailsFieldsComponent', () => {
     expect(endDateInput.value).not.toEqual(startDateInput.value);
     expect(startDateInput.value).toEqual(expectedStartDate);
   });
+
+  it('on get actual date should return the current date', () => {
```

Comment:
I think this test is not that useful. A better test case would be a user trying to change the `start_date` or `end_date` and then,  expecting the app to validate that It can't be a date after the current date.

Reply:
The test is like this because future dates are blocked and the function only gets me the current day.
and the user can only select maximum up to the current date

Reply:
I think it is ok if you want to test function `getActualDate`. But I would recommend to add the other test case.